src/library/scala/collection/convert/impl/ChampStepper.scala

```diff
@@ -157,7 +156,7 @@ extends EfficientSubstep {
   final def substep(): Sub =
     if (!hasNext) null
     else {
-      val root = effectiveRootLevel
+      val root = effectiveRootLevel()
```

Comment:
Without any mutation, I'd have thought the thing to do was remove the parens from the def, but it hardly matters.

Reply:
Thanks, I didn't think much, just went the wrong way